src/components/Home/HomeMap.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import { Menu, Popup, Icon, Label } from 'semantic-ui-react';
+import GoogleMapReact from 'google-map-react';
+
+const style = {
+  borderRadius: 0,
+  border: '1px solid white',
+  opacity: 0.7,
+  backgroundColor: 'grey',
+  color: 'white'
+};
+
+const AnyReactComponent = ({ opened, onClose, labelClass, mouseEnter, mouseLeave, clicked, date, upVote, downVote, city, text, color }) => (
+  <Popup
+    trigger={(
+      <Label
+        as='a'
+        className={labelClass}
+        onClick={clicked}
+        onMouseEnter={mouseEnter}
+        onMouseLeave={mouseLeave}
+        color={color}
+        pointing='below'
+        circular
+      >
+        {text}
+      </Label>
+    )}
+    style={style}
+    size='mini'
+    on='click'
+    open={opened}
+    onClose={onClose}
+  >
+    <Menu.Header>
+      {city}
+    </Menu.Header>
+    <span>Posted: {date}</span>
+    <Menu size='tiny'>
+      <Menu.Item as='a'>
+        <Icon name='thumbs up' />
+        <Label color='teal' floating>
+          {upVote}
+        </Label>
+      </Menu.Item>
+      <Menu.Item as='a'>
+        <Icon name='thumbs down' />
+        <Label color='red' floating>
+          {downVote}
+        </Label>
+      </Menu.Item>
+    </Menu>
+  </Popup>
+);
+
+let today = new Date();
+let dateToday = today.getFullYear() + '-' + (today.getMonth() + 1) + '-' + today.getDate();
+
+class MapHome extends Component {
```

Comment:
Seems like this `MapHome` component only has the render method. Is this going to have its own states and methods in the near future? If so, then this is fine. Otherwise, I would convert this into a functional component.